### PR TITLE
Embed scripts from `kubernetes-conjur-demo` project into `pet-store-d…

### PR DESCRIPTION
**Pull request by moticless**
Adapt and integrate into pet-store-demo (master branch) and eliminate the dependency on kubernetes-conjur-demo. That is, copy only selected scripts of kubernetes-conjur-demo into pet-store-demo.

### CODE REVIEW

File: app-env-DevOperator.yml [hunk 1]
```diff
@@ -66,6 +66,9 @@ spec:
           - name: CONTAINER_MODE
             value: init
 
+          - name: SECRETS_DESTINATION
```
Comment (orenbm) [inline app-env-DevOperator.yml]: This is duplicated. we have it already in line 116. please remove

File: bin/k8s-conjur-deploy/3_init_conjur_cert_authority.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+. utils.sh
+
+announce "Initializing Conjur certificate authority."
+
+set_namespace $CONJUR_NAMESPACE_NAME
+
+conjur_master=$(get_master_pod_name)
+
+if [ $CONJUR_VERSION = '4' ]; then
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/3_init_conjur_cert_authority.sh]: our feature doesn't support conjur v4. we can remove all instances of it

File: bin/k8s-conjur-deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# k8s-conjur-deploy
+This `./start` script deploy k8s Conjur environment by running `./start` of project `kubernetes-conjur-deploy`.
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/README.md]: @sigalsax can you please review the english here?

Comment (orenbm) [reply bin/k8s-conjur-deploy/README.md]: @moticless please don't reference `kubernetes-conjur-deploy` here. that project may change in the future and the code here will not be the same so refernecing there can confuse

File: bin/k8s-conjur-deploy/bootstrap.env [hunk 4]
```diff
@@ -0,0 +1,52 @@
+export CONJUR_VERSION=5
+export CONJUR_APPLIANCE_IMAGE=registry.tld/conjur-appliance:5.0-stable
+export CONJUR_NAMESPACE=[namespace to deploy Conjur to]
+export CONJUR_NAMESPACE_NAME=[namespace to deploy Conjur to]
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/bootstrap.env]: can we merge these 2 into one variable?

File: bin/k8s-conjur-deploy/bootstrap.env [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+export AUTHENTICATOR_ID=[unique authenticator ID for this deployment]
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/bootstrap.env]: please add an example

File: bin/k8s-conjur-deploy/bootstrap.env (hunk 4 above)
Comment (orenbm) [reply bin/k8s-conjur-deploy/bootstrap.env]: also please add an example

File: bin/k8s-conjur-deploy/bootstrap.env [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+export CONJUR_FOLLOWER_COUNT=1
+export PLATFORM=openshift
+export OPENSHIFT_USERNAME=cadmin
+export OPENSHIFT_PASSWORD=Mdm2n7MCMAmzgp2rKft
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/bootstrap.env]: whoa! let's not have this hard-coded here

File: bin/k8s-conjur-deploy/bootstrap.env [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+export OSHIFT_CLUSTER_ADMIN_USERNAME=cadmin
+export OSHIFT_CONJUR_ADMIN_USERNAME=cadmin
+export DOCKER_REGISTRY_PATH=docker-registry-default.openshift-311.itci.conjur.net
+export DEPLOY_MASTER_CLUSTER=true
+export CONJUR_ACCOUNT=[name of conjur account with which to authenticate to seed service]
+export CONJUR_ADMIN_PASSWORD=[conjur admin password]
+
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/bootstrap.env]: can you please have all the variables that need to be updated by the user in one place?

File: bin/k8s-conjur-deploy/bootstrap.env [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+export CONJUR_FOLLOWER_COUNT=1
+export PLATFORM=openshift
+export OPENSHIFT_USERNAME=cadmin
+export OPENSHIFT_PASSWORD=Mdm2n7MCMAmzgp2rKft
+export OSHIFT_CLUSTER_ADMIN_USERNAME=cadmin
+export OSHIFT_CONJUR_ADMIN_USERNAME=cadmin
+export DOCKER_REGISTRY_PATH=docker-registry-default.openshift-311.itci.conjur.net
+export DEPLOY_MASTER_CLUSTER=true
+export CONJUR_ACCOUNT=[name of conjur account with which to authenticate to seed service]
+export CONJUR_ADMIN_PASSWORD=[conjur admin password]
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/bootstrap.env]: this password _will_ be the admin password. please make sure it is understandable that this variables **sets** the password. a user may try to think wether they have a password already

File: bin/k8s-conjur-deploy/bootstrap.env [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+export AUTHENTICATOR_ID=[unique authenticator ID for this deployment]
+export CONJUR_FOLLOWER_COUNT=1
+export PLATFORM=openshift
+export OPENSHIFT_USERNAME=cadmin
+export OPENSHIFT_PASSWORD=Mdm2n7MCMAmzgp2rKft
+export OSHIFT_CLUSTER_ADMIN_USERNAME=cadmin
+export OSHIFT_CONJUR_ADMIN_USERNAME=cadmin
+export DOCKER_REGISTRY_PATH=docker-registry-default.openshift-311.itci.conjur.net
+export DEPLOY_MASTER_CLUSTER=true
+export CONJUR_ACCOUNT=[name of conjur account with which to authenticate to seed service]
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/bootstrap.env]: please make sure it is understandable that we set here the account

File: bin/k8s-conjur-deploy/bootstrap.env [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+export CONJUR_ADMIN_PASSWORD=[conjur admin password]
+
+#######
+
+# Pet Store
+
+#######
+export PROJECT_NAME=pet-store-[username]
+
+
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/bootstrap.env]: remove line

File: bin/k8s-conjur-deploy/bootstrap.env [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+#######
+
+# OPENSHIFT CONFIG
+
+#######
+export PLATFORM=openshift
+export OPENSHIFT_USERNAME=cadmin
+export OSHIFT_CLUSTER_ADMIN_USERNAME=cadmin
+export DOCKER_REGISTRY_PATH=docker-registry-default.openshift-311.itci.conjur.net
+export DEPLOY_MASTER_CLUSTER=true
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/bootstrap.env]: these already exist. please verify you don't have any duplications.

File: bin/k8s-conjur-deploy/policy/templates/project-authn-def.template.yml [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+    description: Identities permitted to authenticate 
+  body:
+  - !layer
+    annotations:
+      description: Layer of authenticator identities permitted to call authn svc
+  - &hosts
+    - !host
+      id: {{ TEST_APP_NAMESPACE_NAME }}/*/*
+      annotations:
+        kubernetes/authentication-container-name: authenticator
```
Comment (orenbm) [inline bin/k8s-conjur-deploy/policy/templates/project-authn-def.template.yml]: please change the container name to `cyberark-secrets-provider`

Comment (orenbm) [reply bin/k8s-conjur-deploy/policy/templates/project-authn-def.template.yml]: also change it in `app-env-DevOperator.yml`

File: bin/k8s-conjur-deploy/3_init_conjur_cert_authority.sh (hunk 2 above)
Comment (moticless) [reply bin/k8s-conjur-deploy/3_init_conjur_cert_authority.sh]: copied file as is. will fix it.

**Pull request merged**